modules/installation-user-infra-machines-iso.adoc

```diff
@@ -4,20 +4,43 @@
 // * installing/installing_restricted_networks/installing-restricted-networks-bare-metal.adoc
 
 [id="installation-user-infra-machines-iso_{context}"]
+ifeval::["{context}" == "installing-ibm-z"]
```

Comment:
Please make a separate module for Z. It looks like only one note is common between the z and not-z sections.

Reply:
Fixed in 53595164cede4ba7699bc02ee397567c6cd74bac